sql/expression/function/conv.go

```diff
@@ -0,0 +1,145 @@
+// Copyright 2022 Dolthub, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package function
+
+import (
+	"fmt"
+	"math"
+	"strconv"
+	"strings"
+
+	"github.com/dolthub/go-mysql-server/sql"
+)
+
+// Conv function converts numbers between different number bases. Returns a string representation of the number N, converted from base from_base to base to_base.
```

Comment:
Nice comment! 😁 This will be helpful for new eyes to our codebase in the future 😉